packages/story-editor/src/components/form/color/color.js

```diff
@@ -100,6 +101,7 @@ const Color = forwardRef(function Color(
     pickerPlacement = PLACEMENT.RIGHT_START,
     isInDesignMenu = false,
     hasInputs = true,
+    width,
```

Comment:
This was being sent in by the [shared color component](https://github.com/GoogleForCreators/web-stories-wp/blob/main/packages/story-editor/src/components/floatingMenu/elements/shared/color.js#L34-L56) inside floating menus, but wasn't ever hooked up. This lead us to get the wrong width in floating menus.